content/pages/2_showcase.html

```diff
@@ -41,9 +41,9 @@ <h2 class="title">
 								</h2>
 								<p>Achieving a precise geometrical representation in numerical simulations typically involves refining mesh elements around curved areas of the geometry. However, this traditional method results in increased memory usage due to the greater number of elements involved.</p>
 								<p>Utilizing polynomials to curve elements to the geometry is a common strategy for enhancing geometrical resolution. Yet, the accuracy of this approach hinges on the degree of the polynomial; insufficient degrees lead to subpar geometrical resolution, even with adaptive mesh refinement techniques.</p>
-								<p>Our innovative approach involves enriching the initial mesh with geometrical data. By leveraging the parametric description of the geometry, we directly interpolate and curve each element to conform precisely to the geometry. Unlike polynomial-based methods, our approach ensures an exact representation of the geometry at every refinement level.</p>
+								<p>Our innovative approach involves enriching the initial mesh with the original geometrical data. By leveraging the parametric description of the geometry, we directly interpolate and curve each element to conform precisely to the geometry. Unlike polynomial-based methods, our approach ensures an exact representation of the geometry at every refinement level.</p>
 								<p>The GIF is a demonstration of this geometry interpolation algorithm in action, showcasing curved tetrahedral mesh elements conforming to an airplane geometry while a refinement plane of level 2 moves along the aircraft's axis.</p>
-								<p>In benchmark tests conducted on a workstation with 100 parallel processes, we achieved remarkable results. Without visualization output, our algorithm processed up to 5.6 billion tetrahedra, with load-balancing taking a mere 7 seconds thanks to the exceptional efficiency of our t8code technology.</p>
+								<p>In benchmark tests conducted on a workstation (768 GB RAM, 12 x 64 GB, DDR4, 2933 MHz) with 100 parallel processes, we achieved remarkable results. Without visualization output, our algorithm processed up to 5.6 billion tetrahedra, with load-balancing taking a mere 7 seconds thanks to the exceptional efficiency of our t8code technology.</p>
```

Comment:
I think this is too much information. If we do not name the proc, then we also do not need the RAM configuration. 768 GB RAM should suffice